disnake/flags.py

```diff
@@ -183,6 +231,29 @@ class SystemChannelFlags(BaseFlags):
         .. describe:: x != y
 
             Checks if two flags are not equal.
+        .. describe:: x | y, x |= y
+
+            Returns a new SystemChannelFlags with all enabled flags from both x and y.
```

Comment:
```suggestion
            Returns a new SystemChannelFlags object with all enabled flags from both x and y.
```
Not sure about this change, could also use "value" or "instance"? The "a SystemChannelFlags" phrasing feels weird